docs/robot/Keywords.html

```diff
@@ -254,17 +254,10 @@ <h2 title='cumulusci.robotframework.CumulusCI'>CumulusCI</h2>
                 </td>
                 <td class="kwargs">
                   
-                  <i>org=None</i>, 
-                  
-                  <i>**userfields</i>
+                  <i>org=None</i>
                   
                 </td>
-                <td class="kwdoc"><p>Returns the login url which will automatically log into the target Salesforce org.  By default, the org_name passed to the library constructor is used but this can be overridden with the org option to log into a different org.</p>
-<p>If userfields are provided, the username and access token for the given user will be used. If not provided, the access token for the org's default user will be used.</p>
-<p>Example:</p>
-<pre>
-${login url}=  Login URL  alias=dadvisor
-</pre></td>
```

Comment:
It looks like we've lost some recent changes to the generated robot keyword docs -- Bryan can tell you how to regenerate it.